server/views/components/interview/index.njk

```diff
@@ -0,0 +1,12 @@
+<div class="interview">
+  {% for qAndA in interview %}
+    <p class="interview__question">
+      <span class="interview__participant">{{ qAndA.question.participant }}</span>
```

Comment:
The [html5 spec](http://w3c.github.io/html/common-idioms-without-dedicated-elements.html#conversations) suggests that this is probably the way to go. Will update to use a `div`.